### DISCUSSION TITLE
Can you change clipskip with diffusers backend?

### DISCUSSION THREAD

**Discussion by a1270** [vladmandic/sdnext#2997] [Q&A] [answered]
I have 'CLIP_stop_at_last_layers' added to my quicksettings and it simply doesn't show up and no setting seems to be present to set it. A model i use requires '-2' and it makes it impossible to use. Even changing it in config.json seems to have no effect.

**Comment by vladmandic** [comment, marked as answer]
clip skip is fully supported and you don't need quicksettings at all, it's already in the main view, just look for it.
And it's muuuch more powerful that old days, you could do clip skip 1.7345 if you want.

**Comment by a1270** [comment]
It's under the 'Advanced' section in the main UI. I have never clicked on that before. Thank you for the answer.
